tests/e2e/utils.go

```diff
@@ -99,6 +100,52 @@ func getActiveApplication() string {
 	return strings.TrimSpace(odoActiveApp)
 }
 
+// returns a local config value of given key and
+// returns an empty string if value is not set
+func findLocalConfigValueOfGivenKey(key string) string {
+	stdOut, _, _ := cmdRunner("odo config view")
+	re := regexp.MustCompile(key + `.+`)
+	odoConfigKeyValue := re.FindString(stdOut)
+	if odoConfigKeyValue == "" {
+		return fmt.Sprintf("%s not found", key)
+	}
+	trimKeyValue := strings.TrimSpace(odoConfigKeyValue)
+	if strings.Compare(key, trimKeyValue) != 0 {
+		return strings.TrimSpace(strings.SplitN(trimKeyValue, " ", 2)[1])
+	}
+	return ""
+}
+
+// returns a global config value of given key and
+// returns an empty string if value is not set
+func findGlobalConfigValueOfGivenKey(key string) string {
```

Comment:
it is not called gloabl config, it should be preferences. How about just `getPreferenceValue`?

Reply:
Done